extension/src/dashboard/pages/Authentication/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Switch, Route, useHistory, useParams } from 'react-router-dom';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+
+import Paper from '@material-ui/core/Paper';
+import MaterialTabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+
+import LoginForm from './LoginForm';
+import SignupForm from './SignupForm';
+
+type TabsProps = {
+  currentTab: string;
+};
+
+const useTabStyle = makeStyles((theme: Theme) => ({
+  tabContainer: {
+    marginBottom: theme.spacing(2),
+  },
+}));
+
+const Tabs: React.FC<TabsProps> = ({ currentTab }) => {
+  const history = useHistory();
+  const classes = useTabStyle({});
+
+  return (
+    <MaterialTabs
+      centered
+      className={classes.tabContainer}
+      value={currentTab}
+      onChange={(_, value) => history.push(`/auth/${value}`)}
```

Comment:
What do you think about a `history.replace` instead?

Reply:
Didn't know about it. Sound better :)